### PR TITLE
add manual confirmation screen in case bot kills itself 

### PR DESCRIPTION
**Pull request by DanyPalma**
image

ouughhhh.... approve my pull request... oughh...

image

### CODE REVIEW

File: apps/admin/src/app/(dashboard)/hackers/page.tsx [hunk 1]
```diff
@@ -382,3 +386,206 @@ export function HackathonsTable() {
...
+      currentIndex > -1 &&
+      currentIndex < allHackers.length - 1
+    ) {
+      setSelectedHacker(allHackers[currentIndex + 1]);
+    } else {
+      setSelectedHacker(null); // No more applicants
+    }
+  };
+
+  type Status =
```
Comment (Lewin-B) [inline apps/admin/src/app/(dashboard)/hackers/page.tsx]: Shouldn't we import this from validators?

**Review by Lewin-B — CHANGES_REQUESTED** [review]
small changes

File: apps/admin/src/app/(dashboard)/hackers/page.tsx (hunk 1 above)
Comment (DanyPalma) [reply apps/admin/src/app/(dashboard)/hackers/page.tsx:447]: ur so right king

Comment (DanyPalma) [reply apps/admin/src/app/(dashboard)/hackers/page.tsx]: See changes (it comes from consts, not validators)

**Review by Lewin-B — APPROVED** [review]
lgtm, once formatting is fixed

**Pull request merged**